grinpy/functions/structural_properties.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+#    Copyright (C) 2017 by
+#    David Amos <somacdivad@gmail.com>
+#    Randy Davila <davilar@uhd.edu>
+#    BSD license.
+#
+# Authors: David Amos <somacdivad@gmail.com>
+#          Randy Davila <davilar@uhd.edu>
+"""Functions for computing various structural properites."""
+
+import networkx as nx
+import itertools
+
+def is_triangle_free(G):
+    """Returns a boolean value according to G containing an induced triangle.
+
+    Parameters
+    ----------
+    G : graph
+        A NetworkX graph.
+
+    Returns
+    -------
+    is_triangle_free : boolean
+                       True if G containes no induced triangle. False if G contains at least one induced triangle.
+
+    Examples
+    --------
+    >>> G = nx.Graph([(0,1),(0,2),(0,3),(1,2),(1,3),(2,3)]) # The complete graph K_4
+    >>> nx.is_triangle_free(G)
```

Comment:
Make sure examples say `gp.is_triangle_free` because `is_triangle_free` is _not_ part of NetworkX. This looks to be the case in all of your examples in the docstrings.